Fragments/Filter/LHE_BGenFilterEQ0_filter.dat

```diff
@@ -0,0 +1,6 @@
+LHEfilter = cms.EDFilter("LHEGenericFilter",
+    src = cms.InputTag("externalLHEProducer"),
+    NumRequired = cms.int32(0),
+    ParticleID = cms.vint32(5),
+    AcceptLogic = cms.string("EQ") # LT meaning < NumRequired, GT >, EQ =, NE !=
+)
```

Comment:
Let's do `LHE_B_EQ0_filter.dat` for now and change everything altogether, removing `_filter.dat`

need to add the chain of modules

Reply:
@agrohsje